src/boot/stylesheets/shared/forms.scss

```diff
@@ -20,72 +9,40 @@
   background-color: $white;
   transition: all .15s ease-in-out;
   box-sizing: border-box;
-  
+
   &::placeholder {
     color: $gray;
   }
-  
+
   &:hover {
     border-color: lighten( $gray, 10% );
   }
-  
+
   &:focus {
     border-color: $blue-wordpress;
     outline: none;
     box-shadow: 0 0 0 2px $blue-light;
-    
+
     &::-ms-clear {
       display: none;
     }
   }
-  
+
   &:disabled {
     background: $white;
     border-color: lighten( $gray, 30% );
     color: lighten( $gray, 30% );
-    
+
     &:hover {
       cursor: default;
     }
-    
+
     &::placeholder {
       color: lighten( $gray, 30% );
     }
   }
 }
 
-%textarea {
+%wpnc-textarea {
   min-height: 92px;
 }
-
-
-// ==========================================================================
```

Comment:
Maybe if we just keep everything in this file as is, but nest it in `wpnc_main`?